cypress/integration/full_run_with_amounts.spec.js

```diff
@@ -0,0 +1,286 @@
+const {
+  checkTableRows,
+  allIncomeRows,
+  getBenefitsBreakdownRows,
+  getAddress,
+} = require('../utils.js')
+
+describe('Full run through', function() {
```

Comment:
I might change the name of the second suite to something more like "Full run through with amounts", so it's immediately obvious which suite is which.